ARCHITECTURE.md

```diff
@@ -60,16 +60,22 @@ Base | CRDs
 Traffic Management | Pilot
 Policy | Policy
 Telemetry | Telemetry
+Telemetry | Prometheus
+Telemetry | Prometheus Operator
+Telemetry | Grafana
+Telemetry | Kiali
+Telemetry | Tracing
 Security | Citadel
 Security | Node agent
 Security | Cert manager
 Configuration management | Galley
 Gateways | Ingress gateway
 Gateways | Egress gateway
 AutoInjection | Sidecar injector
+ThirdParty | CNI
 
 Features and components are defined in the
-[name](https://github.com/istio/operator/blob/905dd84e868a0b88c08d95b7ccf14d085d9a6f6b/pkg/name/name.go#L38) package.
+[name](https://github.com/istio/operator/blob/master/pkg/name/name.go#L246) package.
```

Comment:
Probably not a good idea to use master since this will be changing. You could update the sha to something more recent though.

Reply:
done.